src/services/mocks.ts

```diff
@@ -1,146 +1,140 @@
-  'meshes/default/external-services/httpbin2',
-
-  'meshes/default/dataplanes',
-  'meshes/default/dataplanes/test-dp-02',
-  'meshes/default/dataplanes/ingress-dp-test-123',
-  'meshes/default/dataplanes+insights/ingress-dp-test-123',
-  'meshes/default/dataplanes/gateway-dp-87qntx',
-  'meshes/default/dataplanes/dataplane-test-456',
-  'meshes/default/dataplanes+insights/dataplane-test-456',
-
-  'meshes/default/meshgateways',
-  'meshes/default/meshgateways/edge-gateway',
-
-  'meshes/default/meshgatewayroutes',
-  'meshes/default/meshgatewayroutes/edge-gateway',
-
-  'meshes/default/traffic-traces',
-  'meshes/default/traffic-traces/tt-1',
-  'meshes/default/traffic-traces/tt-3',
-  'meshes/default/traffic-traces/traffic-trace-02',
-  'meshes/hello-world/traffic-traces/tt-123',
-  'meshes/kong-mania-12/traffic-traces/my-silly-mesh-name',
-
-  'meshes/default/health-checks',
-  'meshes/default/health-checks/web-to-backend',
-  'meshes/default/health-checks/web-to-banana',
-  'meshes/default/health-checks/hello-health-check',
-  'meshes/default/health-checks/testing-health-checks',
-  'meshes/default/health-checks/health-check-0023',
-  'meshes/default/health-checks/health-check-12345',
-  'meshes/default/health-checks/foo-bar-baz-123',
-  'meshes/hello-world/health-checks/hello-health-check',
-  'meshes/kong-mania-12/health-checks/testing-health-checks',
-  'meshes/kong-mania-12/health-checks/web-to-banana',
-
-  'meshes/default/fault-injections',
-  'meshes/default/fault-injections/web-to-backend.kuma-system',
-  'meshes/default/fault-injections/fi1.kuma-system',
-
-  'meshes/default/traffic-routes',
-
-  'meshes/default/timeouts',
-
-  'meshes/default/retries',
-
-  'meshes/default/proxytemplates',
-  'meshes/default/proxytemplates/pt-1',
-  'meshes/hello-world/proxytemplates',
-  'meshes/hello-world/proxytemplates/pt-123',
-
-  'meshes/default/traffic-logs',
-  'meshes/default/traffic-logs/tl-1',
-  'meshes/default/traffic-logs/tl-123',
-
-  'meshes/default/traffic-permissions',
-  'meshes/default/traffic-permissions/tp-1',
-  'meshes/default/traffic-permissions/tp-1234',
-  'meshes/default/traffic-permissions/tp-alpha-tango-donut',
-
-  'meshes/default/circuit-breakers',
-  'meshes/default/circuit-breakers/cb1',
-  'meshes/default/circuit-breakers/cb2',
-  'meshes/alpha-tango-mesh/circuit-breakers',
-
-  'zones+insights',
-  'zones+insights/zone-1',
-  'zones+insights/zone-2',
-  'zones+insights/zone-3',
-  'zones+insights/cluster-1',
-  'meshes/default/dataplanes+insights/cluster-1.backend-02',
-  'meshes/default/dataplanes+insights/cluster-1.backend-03',
-  'meshes/default/dataplanes+insights/cluster-1.ingress-02',
-  'meshes/default/dataplanes+insights/cluster-1.gateway-01',
-  'meshes/default/dataplanes+insights/backend',
-  'meshes/default/dataplanes+insights/frontend',
-  'meshes/default/dataplanes+insights/db',
-  'meshes/default/dataplanes+insights/no-subscriptions',
-
-  'global-insights',
-  'policies',
```

Comment:
have you check in any automated way that you've not left out anything here?

Reply:
No, but I’ve changed using my editor’s multi cursor capabilities (i.e. not every line individually by hand).

Previously, this list had 97 entries. Now, it has 104.

9 more because I moved them up (they couldn’t previously be defined there). 1 less because there wasn’t a file for it (meshes/alpha-tango-mesh/circuit-breakers). 1 less because it was defined both in the list and the additional ones below.

97 + 9 - 1 - 1 = 104